drivers/display/Kconfig.mcux_mipi_dsi

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config DISPLAY_MCUX_MIPI_DSI
+	bool "NXP MCUX MIPI DSI driver"
```

Comment:
Why this commit introduces a shim driver instead of a native one for a "well-known" DSI IP?
https://github.com/zephyrproject-rtos/zephyr/pull/36875#issuecomment-1047629683

Reply:
This was added as a first step to prove the functionality. It was deleted in the later commit when we use the MIPI experimental API.